manifests/10_prometheus_rule.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: alert-rules
+    role: alert-rules
+  name: marketplace-alert-rules
+  namespace: openshift-marketplace
+spec:
+  groups:
+    - name: ./marketplace.rules
+      rules:
+        - alert: marketplace-alert
+          expr: rate(opsrc_failure_count[2m]) > 0
```

Comment:
Will need to decide how often to fire an alert.

Reply:
Have you tested this query? I don't see where `opsrc_failure_count` is defined?